### PR TITLE
fix(signature-colleciton): Sign list tweaks

### PR DESCRIPTION
**Pull request by albinagu**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by kksteini — APPROVED** [review]

File: libs/application/ui-shell/src/components/FormExternalDataProvider.tsx [hunk 1]
```diff
@@ -330,6 +330,7 @@ const FormExternalDataProvider: FC<
               }}
               checked={value}
               hasError={error !== undefined}
+              errorMessage={id ? (errors[id] as string) : undefined}
```
Comment (Toti91) [inline libs/application/ui-shell/src/components/FormExternalDataProvider.tsx:333]: What is the reason for this change?

Comment (Toti91) [reply libs/application/ui-shell/src/components/FormExternalDataProvider.tsx:333]: Ah gotcha

**Review by Toti91 — APPROVED** [review]
